pkg/kubeapiserver/options/authentication.go

```diff
@@ -260,6 +261,9 @@ func (s *BuiltInAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 		fs.StringSliceVar(&s.ServiceAccounts.APIAudiences, "service-account-api-audiences", s.ServiceAccounts.APIAudiences, ""+
 			"Identifiers of the API. The service account token authenticator will validate that "+
 			"tokens used against the API are bound to at least one of these audiences.")
+
+		fs.DurationVar(&s.ServiceAccounts.MaxExpTime, "max-service-account-token-expiration", s.ServiceAccounts.MaxExpTime,
+			"The max expiration time of token which user colud request.")
```

Comment:
Say what happens when a larger value is requested. Note that the max is softly enforced.

"The maximum validity duration of a token created by the service account token issuer. If an otherwise valid TokenRequest with a validity duration larger than this value, a token will be issued with a validity duration of this value."